tests/lib/bash/test_header

```diff
@@ -332,7 +332,7 @@ grep_ok() {
     OPTS="$*"
     local TEST_NAME
     TEST_NAME="$(basename "${FILE}")-grep-ok"
-    if grep "${OPTS}" -q -e "${BRE}" "${FILE}"; then
+    if grep ${OPTS} -q -e "${BRE}" "${FILE}"; then
```

Comment:
Change 2/2:

```suggestion
    if grep "${OPTS[@]}" -q -e "${BRE}" ${FILE}"; then
```

Reply:
(Note from Riot discussion, unquoted `${OPTS}` preceded by shellcheck exception comment might be preferable to this, for readability).